.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx --prefer-offline tsdx lint
```

Comment:
npm 7 fundamentally changed the way that scripts are run from other libraries and breaks the way that husky previously worked with the package.json configuration. This is change is documented in the husky docs and putting the command here is what is now required. This still works on npm v6 and yarn v1.
The `--prefer-offline` option prevents tsdx from being downloaded just to run the script but would allow it if the tsdx bin couldn't be found.